### PR TITLE
show the notification text as bubble when sent from webchat

### PR DESCRIPTION
**Pull request by vishnu-ravi**
Trello card: https://trello.com/c/EHhgX0gI/1736-webchat-support-notification-appeareance-options

### CODE REVIEW

File: src/js/yeldaWebchatInjector.js [hunk 1]
```diff
@@ -82,16 +82,17 @@ class YeldaChat {
 
   /**
    * Create an assistantBubbleText component and add it to webChatContainer element
+   * @param {String} text - default null
    */
-  addAssistantBubbleText() {
-    if (!this.webChatContainer || !this.webchatSettings || !this.webchatSettings.bubbleText) {
+  addAssistantBubbleText(text = null) {
+    if (!this.webChatContainer || ((!this.webchatSettings || !this.webchatSettings.bubbleText) && !text)) {
```
Comment (meelie) [inline src/js/yeldaWebchatInjector.js:88]: I would do 

```js
text = text || (webchatSettings && webchatSettings.bubbleText)
```

and then only check that text presence

**Review by meelie — CHANGES_REQUESTED** [review]
Can you please add test to check the innerText is only displayed when bubbleText or notification text ? And properly remove only when needed in both cases ?
Thanks

**Review by codebynao — APPROVED** [review]
Thanks!

**Review by meelie — COMMENTED** [review]
2 additionnal feedback
Thanks !

**Pull request merged**